script.sh

```diff
@@ -312,7 +312,12 @@ EOS
 chmod a+x /usr/sbin/policy-rc.d
 apt-get install --yes --no-install-recommends autopkgtest autodep8
 
-${TRAVIS_DEBIAN_AUTOPKGTEST_RUN} ${TRAVIS_DEBIAN_BUILD_DIR}/*.changes ${TRAVIS_DEBIAN_AUTOPKGTEST_SEPARATOR} null
+TEST_RET=0
+${TRAVIS_DEBIAN_AUTOPKGTEST_RUN} ${TRAVIS_DEBIAN_BUILD_DIR}/*.changes ${TRAVIS_DEBIAN_AUTOPKGTEST_SEPARATOR} null || TEST_RET=$?
+if [ ${TEST_RET} -ne 0 ] && [ ${TEST_RET} -ne 2 ]; then
```

Comment:
please quote these too and the ``then`` should be on a newline without a semicolon.